galleries/examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -63,10 +63,10 @@
 #
 # ``interpolation_stage='data'``: Data -> Interpolate/Resample -> Normalize -> RGBA
 #
-# For both keyword arguments, Matplotlib has a default "antialiased", that is
-# recommended for most situations, and is described below.  Note that this
-# default behaves differently if the image is being down- or up-sampled, as
-# described below.
+# For both keyword arguments, Matplotlib uses a default value of "auto" as
+# specified by the rcParam, which is recommended for most situations.
```

Comment:
If the default is to fall back to the `rcParam`, you should reference it with:
```
:rc:`name.of.rcParam`
```
to get the `rcParam` default automatically inserted.